mediabridge-frontend/src/components/MovieItem.tsx

```diff
@@ -14,23 +28,39 @@ const MovieItem = ({
   movie: Movie;
   onRemove: () => void;
 }) => {
+  const [liked, setLiked] = useState<"up" | "down" | null>(null);
+  const pastelIndex = movie.year % pastelBackgrounds.length;
+  const pastelBg = pastelBackgrounds[pastelIndex];
   return (
     <Card
-      className="flex flex-col items-center mx-2 w-64 text-center break-words"
+      className={`flex flex-col items-center mx-2 w-64 text-center break-words ${pastelBg}`}
       id={`movie-card`}
     >
-      <CardHeader className="w-full px-4">
-        <div className="flex justify-between items-center w-full">
-          <CardTitle className="text-base text-left">{movie.title}</CardTitle>
+      <CardHeader className="w-full px-3 pt-2 pb-2">
+        <div className="flex justify-end items-center gap-2 w-full mb-2">
+          <button
+            type="button"
+            aria-label="Toggle like/dislike"
+            onClick={() => setLiked((prev) => (prev === "up" ? "down" : "up"))}
+            className="p-1 transition-transform hover:scale-110 bg-transparent"
+          >
+            {liked === "down" ? (
+              <ThumbsDown className="w-5 h-5" stroke="red" fill="#f87171" strokeWidth={2} />
+            ) : (
+              <ThumbsUp className="w-5 h-5" stroke="green" fill="#4ade80" strokeWidth={2} />
+            )}
+          </button>
           <button
             onClick={onRemove}
-            className="w-6 h-6 rounded-full bg-red-500 text-white text-xs flex items-center justify-center hover:bg-red-600 transition-colors"
+            className="p-1 hover:scale-110 transition-transform bg-transparent"
```

Comment:
nit: you probably only want this on the left, and you probably want margin, not padding. Try `ml-1`.